src/modules/browser/speech-synthesis/store-module-browser.tsx

```diff
@@ -0,0 +1,40 @@
+import { create } from 'zustand';
+import { persist } from 'zustand/middleware';
+import { useShallow } from 'zustand/react/shallow';
+
+export type BrowsePageTransform = 'html' | 'text' | 'markdown';
```

Comment:
This probably doesn't belong here. We have already a browser store (for the browsing capability) but it's different.